san/get.py

```diff
@@ -1,12 +1,21 @@
 import san.sanbase_graphql
+import operator
 from san.graphql import execute_gql
 from san.pandas_utils import convert_to_datetime_idx_df
 from san.query import get_gql_query, parse_dataset
+from functools import reduce
+
+def get_by_path(root, items):
+    return reduce(operator.getitem, items, root)
```

Comment:
Add documentation and example what this function is doing

Reply:
Did as you asked, but the function is renamed to be "path_to_data" and is moved to the "transform.py" file